frontend/src/common/entities.ts

```diff
@@ -58,6 +58,15 @@ export enum IS_PRIMARY_DATA {
   BOTH = "BOTH",
 }
 
+/**
+ * Author of publication associated with a collection, populated from Crossref as part of collection publication
+ * metadata.
+ */
+export interface Author {
+  family: string;
```

Comment:
ooh I like this over first name last name, since different countries do first name last name differently! 💡